install_functions/installMain.sh

```diff
@@ -213,7 +213,7 @@ function main() {
     if [ -n "${showVersion}" ]; then
         common_logger "Wazuh version: ${wazuh_version}"
         common_logger "Filebeat version: ${filebeat_version}"
-        common_logger "Wazuh installation assistant version: ${wazuh_install_vesion}"
+        common_logger "Wazuh installation assistant revision: ${wazuh_install_revision}"
```

Comment:
```suggestion
        common_logger "Wazuh installation assistant version: ${wazuh_version}-${wazuh_install_revision}"
```

The message should look like this, indicating the product version and revision of the assistant.